major_event_log/templates/major-event-log/index.html

```diff
@@ -6,28 +6,71 @@
 	<div class="page-header">
 		<h1>Major Event Log</h1>
 	</div>
-	<table class="table table-striped table-hover">
-		<thead>
-			<tr>
-				<th>Event</th>
-				<th>Event Date</th>
-				<th>ID</th>
-				<th>Outcome</th>
-			</tr>
-		</thead>
-		<tbody>
-		{% for event in events %}
-			<tr>
-				<td><a href="{% url 'major-event-log:event_details' event.id %}">{{ event.title }}</a></td>
-				<td>{{ event.date }}</td>
-				<td>{{ event.id }}</td>
-				{% if event.get_outcome_display == 'Success' %}
-				<td><span class="label label-success">{{ event.get_outcome_display }}</span></td>
-				{% else %}
-				<td><span class="label label-danger">{{ event.get_outcome_display }}</span></td>
-				{% endif %}
-			</tr>
-		{% endfor %}
-		</tbody>
-	</table>
+    {% if events %}
+        <table class="table table-striped table-hover">
+            <thead>
+                <tr>
+                    <th>ID</th>
+                    <th>Event Date</th>
+                    <th>Event</th>
+                    <th>Outcome</th>
+                </tr>
+            </thead>
+            <tbody>
+            {% for event in events %}
+                <tr>
+                    <td class="monospaced"><a href="{% url 'major-event-log:event_details' event.id %}">{{ event.id|truncatechars:16 }}</a></td>
+                    <td>{{ event.date }}</td>
+                    <td>{{ event.title }}</td>
+                    {% if event.get_outcome_display == 'Success' %}
+                    <td class="monospaced"><span class="label label-success">{{ event.get_outcome_display }}</span></td>
+                    {% else %}
+                    <td class="monospaced"><span class="label label-danger">{{ event.get_outcome_display }}</span></td>
+                    {% endif %}
+                </tr>
+            {% endfor %}
+            </tbody>
+        </table>
+        {% if is_paginated %}
```

Comment:
Ah, yes, I noticed others did that too. I wasn't sure it was justified in this case because, as you mentioned, it's only used in one template, but I see your reasoning for preferring it separated. That's easy enough to do.